context.go

```diff
@@ -252,6 +274,39 @@ func (bc *beanContext) json(code int, i interface{}, indent string) error {
 	return enc.Encode(i)
 }
 
+func (bc *beanContext) jsonp(code int, i any, indent string, charset ...string) (err error) {
+	if len(charset) > 0 {
+		bc.writeContentType(jsonpCTypeUTF8 + ";" + charset[0])
+	} else {
+		bc.writeContentType(jsonpCTypeUTF8 + ";" + "charset=utf-8")
+	}
+	bc.response.WriteHeader(code)
+
+	callback := bc.Query("callback")
```

Comment:
Is' pretty 'and' callback 'a reserved keyword similar to beanContext?

Reply:
`pretty` is not a reserved keyword but callback is for jsonp return.